src/python/grpcio/grpc/_server.py

```diff
@@ -65,12 +65,23 @@ def _serialized_request(request_event):
   return request_event.batch_operations[0].received_message.bytes()
 
 
-def _code(state):
```

Comment:
Good question!

Short answer: the former.

Long answer: these changes make safe service-side application code that fails not by raising an exception but by merely returning an invalid value (like None). Putting the fix in `_server.py` means that the fix applies to such illegitimate code written against both APIs.